frontend/src/store/thunks.ts

```diff
@@ -985,7 +985,7 @@ export const reportBadMergeAsync = (dispatch: Dispatch) => async () => {
 
 export const fetchCalendarAsync = (dispatch: Dispatch) => async (
   teamID: TeamID,
-  month = new Date()
+  month: number
```

Comment:
I think it would be good to suffix these dates with something like `_timestamp` or `_ts` as `month` could be `1..12`